executor/executor.spec

```
@@ -0,0 +1,101 @@
+    'uvicorn.protocols',
+    'uvicorn.protocols.http',
+    'uvicorn.protocols.http.auto',
+    'uvicorn.protocols.websockets',
+    'uvicorn.protocols.websockets.auto',
+    'uvicorn.lifespan',
+    'uvicorn.lifespan.on',
+]
+
+# Collect package data
+for package in ['fastapi', 'uvicorn', 'pydantic', 'anthropic', 'claude_agent_sdk',
+                'agno', 'openai', 'mcp', 'sqlalchemy', 'httpx']:
```

RuboCop (1.81.7)

[fatal] unexpected token tCOLON
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)